### PR TITLE
[Hackadoc] Op Insights - Azure Storage topic

### PR DESCRIPTION
**Pull request by bandersmsft**
New topic written by Jim Parker

### CODE REVIEW

**Comment by mollybostic** [comment]
@bandersmsft is updating title and description to clarify that the topic is about Op Insights + Storage, and also adding Next Steps section.

**Comment by bandersmsft** [comment]
Updated the article for changes that Molly said were needed.

File: articles/operational-Insights-connect-azure-storage.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+<tags
+   ms.service="Operational-Insights"
+   ms.devlang="may be required"
+   ms.topic="article"
+   ms.tgt_pltfrm="may be required"
+   ms.workload="Operational-Insights" 
+   ms.date="02/20/2015"
+   ms.author="jimpark"/>
+
+# Connect Azure Storage
```
Comment (mollybostic) [inline articles/operational-Insights-connect-azure-storage.md]: Please add Operational Insights to the h1 title as well

File: articles/operational-Insights-connect-azure-storage.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<properties 
+   pageTitle="Connect Operational Insights to Azure Storage" 
+   description="Connecting Operational Insights to Azure Storage enables you to collect diagnostic telemetry data from a worker role, web role, or virtual machine running in Azure." 
+   documentationCenter="dev-center-name" 
+   authors="Jim-Parker" 
+   manager="jwhit" 
+   editor=""/>
+
+<tags
+   ms.service="Operational-Insights"
```
Comment (mollybostic) [inline articles/operational-Insights-connect-azure-storage.md]: value needs to be all lowercase

**Comment by mollybostic** [comment]
closing for now, Bill will update.

**Pull request closed without merging**